CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## Your new git-extra command should support
+
+* OSX, Ubuntu, FreeBsd(You may need to browse their man page)
+* Bash 3.2+ (If you are not sure, [Bash versions](http://tldp.org/LDP/abs/html/bash2.html))
```

Comment:
Actually, that link goes to an unofficial changelog between the versions of Bash. There may be inaccuracies there.

The documentation is part of Advanced Bash Scripting Guide, which was written by a guy who wanted to learn to script with bash, and so it's pointed out regularly around the web that it's not the best resource for everything. It is a very good guide, and it has received a lot of feedback and suggestions from people like Stéphane Chazelas, who seems to know everything there is to know about bash and related topics.

Anyways, I think I'd prefer it'd said `Bash 3.2+ (If you aren't sure, see [the bash changelog])`.